Source/player.cpp

```diff
@@ -501,7 +502,7 @@ player_graphic GetPlayerGraphicForSpell(spell_id spellId)
 	}
 }
 
-void StartSpell(int pnum, Direction d, int cx, int cy)

```

Comment:
This looks wrong. Should probably be:
```cpp
void StartSpell(int pnum, Direction d, WorldTileCoord position)
```

Reply:
I'm not sure what you mean, I've simply changed `int` to `WorldTileCoord` to remove the need for `static_cast` inside the function

Reply:
arh, i didn't notice that there where two types here.